apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/GoogleLogger.cs

```diff
@@ -151,7 +152,7 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
             while (currentLogScope != null)
             {
                 // Determine if the state of the scope are format params
-                if (currentLogScope.State is IEnumerable<KeyValuePair<string, object>> scopeFormatParams)
+                if (currentLogScope.State is FormattedLogValues scopeFormatParams)
```

Comment:
There are two other scopes that are created automatically by Asp.Net Core workflow whose states are also of type `IEnumerable<KeyValuePair<string, object>>` although they are not format params. They were creeping in `parent_scopes` (as seen on the snippet below) and the test was failing because it was correctly expecting one format params parent scope.
```
parent_scopes: [
    0: {
        id:  "20180903t111549053"     
        {OriginalFormat}:  "ScopeFormatParameters - {id}"     
    }
    1: {
        ActionId:  "c464c392-2ea9-43a8-b7fe-8350c0bbff77"     
        ActionName:  "Google.Cloud.Diagnostics.AspNetCore.IntegrationTests.MainController.ScopeFormatParameters (Google.Cloud.Diagnostics.AspNetCore.IntegrationTests)"     
    }
    2: {
        RequestId:  "0HLGHHQDDN916"     
        RequestPath:  "/Main/ScopeFormatParameters/20180903t111549053"     
    }
]
```

Reply:
Makes sense, thanks for fixing!